### DISCUSSION TITLE
There is anyway to change the item stack display font color or add outline or shadow

### DISCUSSION THREAD

**Discussion by enemy141** [peter-kish/gloot#233] [Q&A] [answered]
Hi I have been learn gloot for quite a while now So there is any solution to alter the theme front color ??
![image](https://github.com/user-attachments/assets/11355088-99ab-45a5-881f-56f86372a2f6)

**Comment by peter-kish** [comment, marked as answer]
The numbers displaying stack sizes are basically just labels "under the hood".
So creating a theme for the control and setting the label font/color inside it should do the trick.
